includes/class-newspack-ads-gam.php

```diff
@@ -495,16 +495,37 @@ function( $key ) {
 		return [];
 	}
 
+	/**
+	 * Verify WP environment to make sure it's safe to use GAM.
+	 * 
+	 * @return bool Wether it's safe to use GAM.
+	 */
+	public static function is_sdk_compatible() {
```

Comment:
Nit: not sure about the name, I'd rather expect something like `is_environment_compatible`. In the future this might also check e.g. PHP version.